pkgs/by-name/ya/yascreen/package.nix

```
@@ -3,35 +3,28 @@
   stdenv,
   go-md2man,
   fetchFromGitHub,
-  fetchpatch,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "yascreen";
-  version = "1.86";
+  version = "2.06";
 
   src = fetchFromGitHub {
     owner = "bbonev";
     repo = "yascreen";
-    rev = "v${version}";
-    sha256 = "sha256-z7j2yceiUyJNdyoVXAPiINln2/MUMqVJh+VwQnmzO2A=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-CIeWghtOnuQmEMwIpC1Xo1zLsuV4c0V7YAXTqUAzX1g=";
   };
 
   nativeBuildInputs = [ go-md2man ];
-  makeFlags = [ "PREFIX=$(out)" ];
 
-  patches = [
-    (fetchpatch {
-      url = "https://github.com/bbonev/yascreen/commit/a30b8fce66a3db9f1194fede30a48424ed3d696b.patch";
-      sha256 = "sha256-Bnaf3OVMlqyYMdGsJ6fF3oYsWT01FcjuRzxi6xfbnZg=";
-    })
-  ];
+  makeFlags = [ "PREFIX=$(out)" ];
 
-  meta = with lib; {
+  meta = {
     homepage = "https://github.com/bbonev/yascreen";
     description = "Yet Another Screen Library (curses replacement for daemons and embedded apps)";
```

```suggestion
    description = "Curses replacement for daemons and embedded apps";
```
> meta.description must not start with the package name

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes